templates/listing-page-filters.twig

```diff
@@ -14,4 +14,17 @@
             </option>
         {% endfor %}
     </select>
+    <select class="form-select listing-page-select" id="post-type">
+        <option value="">{{__( 'All posts', 'planet4-master-theme' ) }}</option>
+        {% for key,post_type in post_types %}
+            {% set isSelected = current_post_type == post_type.slug %}
+            <option
+                value="{{post_type.slug}}"
+                {{isSelected ? 'selected' : ''}}
```

Comment:
I think in this case could be better if we keep the attribute with the proper boolean.
```suggestion
    selected={{isSelected}}
```
What do you think?

Reply:
This doesn't work, it results in this:

<img width="472" alt="Screenshot 2024-12-17 at 15 20 56" src="https://github.com/user-attachments/assets/d02abaea-a619-41ad-8fba-83d196be9d00" />